lib/scope.js

```diff
@@ -1401,8 +1401,7 @@ module.exports = {
       let elem = document.querySelector( selector );
       // If the element has a sibling that's a label, we need to
       // interact with that label instead.
-      // `elem` has never been null from what we've seen
-      if ( elem.nextSibling && elem.nextSibling.tag === `label` ) {
+      if ( elem && elem.nextSibling && elem.nextSibling.tag === `label` ) {
```

Comment:
Log non-existent elems for debugging: Not sure how to add a debug log outside of the DOM here, but we should have some kind of response to an element not existing, I think. Or we shouldn't send selectors for elements that don't exist. I'm not sure how that's even happening. Maybe we can send in `reports` to add to or some such.

Reply:
Made https://github.com/SuffolkLITLab/ALKiln/issues/808